int8    radar_pos_id
float32 distance

